stdlib/http/src/main/ballerina/src/http/client_endpoint.bal

```diff
@@ -157,7 +214,14 @@ public type Client client object {
     # + request - An HTTP inbound request message
     # + return - The response for the request or an `http:ClientError` if failed to establish communication with the upstream server
     public remote function forward(@untainted string path, Request request) returns Response|ClientError {
-        return self.httpClient->forward(path, request);
+        Response|ClientError response = self.httpClient->forward(path, request);
+        if (response is Response) {
+            error? x = addObservabilityInformation(path, HTTP_FORWARD, response.statusCode);
```

Comment:
The HTTP method should be extraced from the request. This is simply a remote function to forward a request

Reply:
Fixed with 527d3559b6c04d5002739eaffbabbfa83ea6856a